backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -1,194 +1,250 @@
-<div data-hook="admin_product_form_fields">
-
-  <div class="row">
-
-    <div class="left col-xs-9" data-hook="admin_product_form_left">
-      <div data-hook="admin_product_form_name">
-        <%= f.field_container :name do %>
-          <%= f.label :name, class: 'required' %>
-          <%= f.text_field :name, :class => 'fullwidth title', :required => true %>
-          <%= f.error_message_on :name %>
-        <% end %>
-      </div>
-
-      <div data-hook="admin_product_form_slug">
-        <%= f.field_container :slug do %>
-          <%= f.label :slug, class: 'required' %>
-          <%= f.text_field :slug, :class => 'fullwidth title', :required => true %>
-          <%= f.error_message_on :slug %>
-        <% end %>
-      </div>
-
-      <div data-hook="admin_product_form_description">
-        <%= f.field_container :description do %>
-          <%= f.label :description %>
-          <%= f.text_area :description, {:rows => "#{unless @product.has_variants? then '22' else '15' end}", :class => 'fullwidth'} %>
-          <%= f.error_message_on :description %>
-        <% end %>
-      </div>
-    </div>
-
-    <div class="right col-xs-3" data-hook="admin_product_form_right">
-      <div data-hook="admin_product_form_price">
-        <%= f.field_container :price do %>
-          <%= f.label :price, class: 'required' %>
-          <%= f.text_field :price, :value => number_to_currency(@product.price, :unit => ''), :required => true %>
-          <%= f.error_message_on :price %>
+<div class="container-fluid">
+  <div data-hook="admin_product_form_fields" class="row">
+    <div data-hook="admin_product_form_left" class="col-lg-8 col-xs-12">
+
+      <div class="row form-section">
```

Comment:
I notices that this has CSS applied strangely.

`.row` as defined by bootstrap has a margin left and right of `-0.9375rem`, which patches a padding of `0.9375rem` on the columns.

`.form-section` has `margin: 10px 0px`.

So adding these two together adds padding in a non-obvious way. We instead probably want to add some padding to `.form-section` and avoid using `row` here at all.